packages/opentelemetry-metrics/src/types.ts

```diff
@@ -54,3 +48,9 @@ export const DEFAULT_METRIC_OPTIONS = {
   unit: '1',
   valueType: api.ValueType.DOUBLE,
 };
+
+/** Represents an metrics collection interface. */
+export interface MetricsCollector {
```

Comment:
Is there some reason you decided to make this its own collector interface instead of just adding it to the existing meter provider?